### PR TITLE
Cosmin Herciu-Sava branch

### PR DESCRIPTION
**Pull request by CosminHerciuSava**
Added firstname lastname tasks.

### CODE REVIEW

File: CosminHerciuSava-dir/CosminHerciuSava.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This file was created by Cosmin Herciu-Sava.
+second commit
```
Comment (raduchistest) [inline CosminHerciuSava-dir/CosminHerciuSava.txt:2]: please add "reviewed by radu chis"

File: CosminHerciuSava-dir/OpenMPHelloWorld.cpp [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#include <omp.h>
+
+#include <stdio.h>
+#include <stdlib.h>
+
+int main(int argc, char* argv[])
+{
+	#pragma omp parallel
+	{
+		printf("Hello World... from thread = %d\n", omp_get_thread_num());
```
Comment (raduchistest) [inline CosminHerciuSava-dir/OpenMPHelloWorld.cpp:10]: some more comments here

**Review by raduchistest — CHANGES_REQUESTED** [review]

**Review by raduchistest — APPROVED** [review]

File: CosminHerciuSava-dir/CosminHerciuSava.txt (hunk 1 above)
Comment (claudius2) [inline CosminHerciuSava-dir/CosminHerciuSava.txt:2]: Please add "reviewed by claudiu stefan"

File: CosminHerciuSava-dir/OpenMPHelloWorld.cpp (hunk 2 above)
Comment (raduchistest) [reply CosminHerciuSava-dir/OpenMPHelloWorld.cpp:10]: ok!

File: CosminHerciuSava-dir/CosminHerciuSava.txt (hunk 1 above)
Comment (raduchistest) [reply CosminHerciuSava-dir/CosminHerciuSava.txt:2]: ok

**Pull request merged**